test/e2e/es/mutation_test.go

```diff
@@ -108,7 +108,7 @@ func TestMutationResizeMemoryUp(t *testing.T) {
 				corev1.ResourceMemory: resource.MustParse("4Gi"),
 				corev1.ResourceCPU:    resource.MustParse("2"),
 			},
-		})
+		}).TolerateMutationChecksFailures()
```

Comment:
Let's maybe add a comment with a link to https://github.com/elastic/cloud-on-k8s/issues/8267 to make it easier to understand why we made this choice.

Reply:
Yes! I forget good habits. Thanks.